src/main/groovy/io/seqera/wave/service/aws/S3ClientFactory.groovy

```diff
@@ -38,13 +40,16 @@ import software.amazon.awssdk.services.s3.S3Client
 @Factory
 @CompileStatic
 @Slf4j
-@Requires(property = 'wave.blobCache.enabled', value = 'true')
 class S3ClientFactory {
 
     @Inject
     private BlobCacheConfig blobConfig
 
+    @Value('${aws.region}')
```

Comment:
I believe this is a new setting, isn't it?

Reply:
ah ok, is defined here 

https://github.com/seqeralabs/wave/blob/6ffb091daa884f3a3b778cd22afa229b10f7ccc3/src/main/resources/bootstrap.yml#L4-L5